addons/survey/models/survey_user.py

```diff
@@ -25,7 +25,7 @@ class SurveyUserInput(models.Model):
     deadline = fields.Datetime('Deadline', help="Datetime until customer can open the survey and submit answers")
     state = fields.Selection([
         ('new', 'Not started yet'),
-        ('skip', 'Partially completed'),
+        ('in_progress', 'In Progress'),
```

Comment:
Commit3: maybe you will hate me and/or ignore this, but couldn't we find something smaller than in_progress, considering that billions of users will take our surveys ? like ongoing. Well saving 4 letters is maybe not much. Boarf. I let this comment because I typed it. Bisous.

Reply:
Now that we have renamed some templates to xxx_in_progress, I can no longer make it happen (or I can but...). Sorry 😇